{"name": "错误账号密码", "status": "passed", "description": "这是测试用例用例2的描述信息", "start": 1625213018122, "stop": 1625213018126, "uuid": "0e847f08-4b21-4adf-8019-1f2f41e2287f", "historyId": "9078c34ff831bb74ca4c0c76bfe75e29", "testCaseId": "92e7901b1cae8655b4042e7eeb0890dd", "fullName": "testCase.test.Testcase#test_case02", "labels": [{"name": "feature", "value": "登录模块"}, {"name": "story", "value": "登录模块下的子模块:错误登录账号密码"}, {"name": "parentSuite", "value": "testCase"}, {"name": "suite", "value": "test"}, {"name": "subSuite", "value": "Testcase"}, {"name": "host", "value": "XSD-20190315NRK"}, {"name": "thread", "value": "16880-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCase.test"}]}